github/event_types.go

```diff
@@ -345,6 +345,42 @@ type PingEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// ProjectEvent is triggered when project is created, modified or deleted.
+// the webhook event name is "project".
+//
+// GitHub docs https://developer.github.com/v3/activity/events/types/#projectevent
+type ProjectEvent struct {
+	Action        *string       `json:"action,omitempty"`
+	Project       *Project      `json:"project,omitempty"`
+	Repository    *Repository   `json:"repository,omitempty"`
+	Organizaition *Organization `json:"organization,omitempty"`
```

Comment:
All other events seem to rename this field to `Org` rather than `Organization`. I think we should do the same here, for consistency.